src/App.vue

```diff
@@ -52,7 +49,7 @@ button {
   @apply after:right-4 after:bottom-1/2 after:translate-y-1/2;
 }
 
-[tabindex="0"] {
+[tabindex='0'] {
```

Comment:
I actually didn't like the single quotes in CSS, that's why I didn't do anything about it.

But I guess, you'll tell me I'm the one who chose the single quotes and now I have to be consistent 😛